v3/src/components/graph/models/graph-controller.ts

```diff
@@ -62,22 +62,25 @@ export class GraphController {
   initializeGraph() {
     const {graphModel, layout, dotsRef, enableAnimation, instanceId, v2Document} = this,
       dataConfig = graphModel.config
+
     if (v2Document) {
       this.processV2Document()
     } else {
-      graphModel.setGraphProperties({
-        axes: {bottom: EmptyAxisModel.create({place: 'bottom'}),
+      if (!dotsRef.current){
+        graphModel.setGraphProperties({
+          axes: {bottom: EmptyAxisModel.create({place: 'bottom'}),
           left: EmptyAxisModel.create({place: 'left'})}, plotType: 'casePlot'
-      })
+        })
+      }
+      else if (dotsRef.current) {
```

Comment:
```suggestion
      else {
```
don't need the condition